apps/web/app/components/title.tsx

```diff
@@ -0,0 +1,11 @@
+import { BoxProps, styled } from 'leather-styles/jsx';
+
+type TitleProps = BoxProps;
+
+export function Title({ children, ...props }: TitleProps) {
```

Comment:
What is this title, and how does it differ from `PageHeader`?

Reply:
removed